ocs_ci/ocs/monitoring.py

```diff
@@ -227,3 +473,30 @@ def prometheus_health_check(name=constants.MONITORING, kind=constants.CLUSTER_OP
 
     logging.error(f"Prometheus cluster is degraded {health_conditions}")
     return False
+
+
+def check_ceph_metrics_available():
```

Comment:
This function is reimplementing OCS-1268, which is already covered in https://github.com/red-hat-storage/ocs-ci/blob/19605e18c5a5cdc77ca454b01ea26343cf1ab7a4/tests/manage/monitoring/prometheusmetrics/test_monitoring_defaults.py#L128

But it:
- doesn't take into account that on a cloud platforms, not all metrics are available (because RGW is not deployed there) or it doesn't cover all metrics of the platform (I haven't checked which what is the case)
-  doesn't properly log all metrics it asks propetheus instance for (this could be very useful, when inspecting what was actually exactly tested)

Reply:
Witch changes from https://github.com/red-hat-storage/ocs-ci/pull/2798, you can use list of metrics and basic check function from metrics module to use standard logging and avoid duplication.